registration.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<title>good.teens</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
Don't seen need for jQuery on the site or bootstrap on the site.

Reply:
Was reading https://www.swyx.io/writing/js-third-age/ this morning. Made me think back to the jQuery days. I would focus on new tech as you continue to expand your JavaScript skills.